# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/windows/msaa/nsEventMap.h
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/windows/msaa/nsEventMap.h
--- a/accessible/windows/msaa/nsEventMap.h	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/windows/msaa/nsEventMap.h	Wed Nov 29 03:53:20 2023 +0000
@@ -47,7 +47,6 @@
   kEVENT_WIN_UNKNOWN,                                // nsIAccessibleEvent::EVENT_WINDOW_MINIMIZE
   kEVENT_WIN_UNKNOWN,                                // nsIAccessibleEvent::EVENT_WINDOW_RESTORE
   IA2_EVENT_OBJECT_ATTRIBUTE_CHANGED,                // nsIAccessibleEvent::EVENT_OBJECT_ATTRIBUTE_CHANGED
-  kEVENT_WIN_UNKNOWN,                                // nsIAccessibleEvent::EVENT_VIRTUALCURSOR_CHANGED
   EVENT_OBJECT_VALUECHANGE,                          // nsIAccessibleEvent::EVENT_TEXT_VALUE_CHANGE
   kEVENT_WIN_UNKNOWN,                                // nsIAccessibleEvent::EVENT_SCROLLING
   kEVENT_WIN_UNKNOWN,                                // nsIAccessibleEvent::EVENT_ANNOUNCEMENT